deployment/services/graphql.ts

```diff
@@ -134,8 +134,14 @@ export function deployGraphQL({
           GRAPHQL_PUBLIC_ORIGIN: `https://${environment.appDns}`,
           CDN_CF: '1',
           HIVE_USAGE: '1',
-          HIVE_USAGE_TARGET: hiveConfig.require('target'),
+          HIVE_TARGET: hiveConfig.require('target'),
           HIVE_USAGE_ENDPOINT: serviceLocalEndpoint(usage.service),
+          HIVE_TRACING: '1',
+          HIVE_TRACING_ENDPOINT: environment.isProduction
```

Comment:
note: using the public endpoints to avoid having to solve a circular pulumi dependency of graphql api and otel :D We can alter on fix it.